### PR TITLE
Changes made to tests and verification_util to handle the error valid…

### PR DESCRIPTION
**Pull request by manojsalunke85**
…ation messages fixed

Signed-off-by: manoj <manoj.salunke@intel.com>

### CODE REVIEW

File: tests/automation_framework/tests/work_order_tests/test_submit.py [hunk 1]
```diff
@@ -529,19 +499,18 @@ def test_work_order_signing_wrong(self):
 
         result_response = self.test_obj.getresult(
```
Comment (jinengandhi-intel) [inline tests/automation_framework/tests/work_order_tests/test_submit.py:470]: For the work_order_signing_wrong, the test case was failing and now it's passing? I see that you have changed the assert from check_negative_response to verify_test

File: tests/automation_framework/tests/work_order_tests/test_submit.py [hunk 2]
```diff
@@ -566,18 +535,18 @@ def test_work_order_workerEncryptionKey_special_character(self):
...
             self.test_obj.build_request_output['request_obj'])
 
-        # assert (
-        #         verify_test(
-        #             result_response, 0,
-        #             self.test_obj.build_request_output['pre_test_output'],
-        #             self.test_obj.build_request_output['action_obj'])
-        #         is TestStep.SUCCESS.value)
-
         assert (
```
Comment (jinengandhi-intel) [inline tests/automation_framework/tests/work_order_tests/test_submit.py:502]: Similar to above comment, the assert got changed from check_negative to verify_test.

File: tests/automation_framework/tests/work_order_tests/test_submit.py [hunk 3]
```diff
@@ -633,14 +602,14 @@ def test_work_order_with_alternate_dataEncryption_algorithm (self):
             self.test_obj.build_request_output['request_obj'])
 
         assert (
-                verify_test(
-                    result_response, 0,
-                    self.test_obj.build_request_output['pre_test_output'],
-                    self.test_obj.build_request_output['action_obj'])
+                check_negative_test_responses(
```
Comment (jinengandhi-intel) [inline tests/automation_framework/tests/work_order_tests/test_submit.py:564]: Here it changed from verify_test to check_negative.

File: tests/automation_framework/tests/work_order_tests/test_submit.py [hunk 4]
```diff
@@ -782,10 +752,9 @@ def test_work_order_with_no_indata (self):
...
```
Comment (jinengandhi-intel) [inline tests/automation_framework/tests/work_order_tests/test_submit.py:717]: Similar to above.

File: tests/automation_framework/tests/work_order_tests/test_submit.py [hunk 5]
```diff
@@ -1141,10 +1116,9 @@ def test_work_order_submit_requesterId_param_remove (self):
...
```
Comment (jinengandhi-intel) [inline tests/automation_framework/tests/work_order_tests/test_submit.py:1089]: Similar to above

File: tests/automation_framework/tests/work_order_tests/test_submit.py [hunk 6]
```diff
@@ -1171,9 +1145,8 @@ def test_work_order_with_response_timeout_str (self):
...
```
Comment (jinengandhi-intel) [inline tests/automation_framework/tests/work_order_tests/test_submit.py:1118]: Similar to above.

File: tests/automation_framework/tests/work_order_tests/test_submit.py [hunk 7]
```diff
@@ -566,18 +535,18 @@ def test_work_order_workerEncryptionKey_special_character(self):
...
-                check_negative_test_responses(
-                    submit_response,
-                    "Invalid Request")
+                verify_test(
+                    result_response, 0,
+                    self.test_obj.build_request_output['pre_test_output'],
+                    self.test_obj.build_request_output['action_obj'])
                 is TestStep.SUCCESS.value)
+
+        # assert (
```
Comment (jinengandhi-intel) [inline tests/automation_framework/tests/work_order_tests/test_submit.py]: Remove the commented code if not needed

**Review by jinengandhi-intel — CHANGES_REQUESTED** [review]
Manoj can you look at the comments and address them.

File: tests/automation_framework/tests/work_order_tests/test_submit.py (hunk 3 above)
Comment (manojsalunke85) [reply tests/automation_framework/tests/work_order_tests/test_submit.py:564]: This is the right change to handle the error. earlier it was verify test and now the issue has been fixed and it gives the result as "Unsupported dataEncryptionAlgorithm found in the request"

File: tests/automation_framework/tests/work_order_tests/test_submit.py (hunk 4 above)
Comment (manojsalunke85) [reply tests/automation_framework/tests/work_order_tests/test_submit.py:717]: This is the right change to handle the error. earlier it was verify test and now the issue has been fixed and it gives the result as "Missing parameter inData"

File: tests/automation_framework/tests/work_order_tests/test_submit.py (hunk 5 above)
Comment (manojsalunke85) [reply tests/automation_framework/tests/work_order_tests/test_submit.py:1089]: "Missing parameter requesterId" is the new error introduced by Ram. Hence the change is valid.

File: tests/automation_framework/tests/work_order_tests/test_submit.py (hunk 6 above)
Comment (manojsalunke85) [reply tests/automation_framework/tests/work_order_tests/test_submit.py:1118]: "Invalid data format for responseTimeoutMSecs" is the new error introduced by Ram. hence change is valid

File: tests/automation_framework/tests/work_order_tests/test_submit.py (hunk 1 above)
Comment (manojsalunke85) [reply tests/automation_framework/tests/work_order_tests/test_submit.py:470]: Reverting back to check_negative_test as i have to revisit the test to get confirmation from Ram on the behavior.

File: tests/automation_framework/tests/work_order_tests/test_submit.py (hunk 2 above)
Comment (manojsalunke85) [reply tests/automation_framework/tests/work_order_tests/test_submit.py:502]: Reverting back to check_negative_test as i have to revisit the test to get confirmation from Ram on the behavior.

File: tests/automation_framework/tests/work_order_tests/test_submit.py (hunk 1 above)
Comment (manojsalunke85) [reply tests/automation_framework/tests/work_order_tests/test_submit.py:470]: changes addressed in commit "3010f9e43d2d6a3cea2f19a464bd6efbae48e964"

File: tests/automation_framework/tests/work_order_tests/test_submit.py (hunk 2 above)
Comment (manojsalunke85) [reply tests/automation_framework/tests/work_order_tests/test_submit.py:502]: changes addressed in commit "3010f9e43d2d6a3cea2f19a464bd6efbae48e964"

File: tests/automation_framework/tests/work_order_tests/test_submit.py (hunk 7 above)
Comment (manojsalunke85) [reply tests/automation_framework/tests/work_order_tests/test_submit.py]: changes addressed in commit "3010f9e43d2d6a3cea2f19a464bd6efbae48e964"

**Review by jinengandhi-intel — APPROVED** [review]

**Pull request merged**